app/views/newflow/signup/_change_signup_email_form_template.html.erb

```diff
@@ -36,9 +40,13 @@
                 </div>
 
                 <div class="content">
-                    <%= f.submit I18n.t(:'login_signup_form.send_my_pin'),
-                                        disable_with: I18n.t(:'login_signup_form.continue_button'),
-                                        class: 'primary' %>
+                    <%=
+                        f.submit(
+                            I18n.t(:"login_signup_form.send_my_pin"),
+                            disable_with: I18n.t(:"login_signup_form.continue_button"),
+                            class: 'primary'
```

Comment:
would like to see consistent use of " vs '.   i like the idea of ' for non-interpolated strings, " for interpolated ones.  this allows a quick glance to know which is which.

Reply:
You're referring to the translations' strings? I intentionally made the exception to that rule for ALL translations. At least now, though, that rule is consistent across ALL translations whereas before very very few of them used single quotes for translations. My personal preference when it comes to translations, so it could be changed. It would just be a TON of changes.